### PR TITLE
Pioneer 16 Glider Cals

### PR DESCRIPTION
**Pull request by cdobs**
Adds calibration CSVs for sensors to be deployed on P16 gliders

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09295__20200217.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9295,CC_calibration_date,1581897600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09295__20200217.csv:2]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09301__20201001.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9301,CC_calibration_date,1601510400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09301__20201001.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00432__20210208.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+432,CC_calibration_date,1612742400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00432__20210208.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00539__20210208.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+539,CC_calibration_date,1612742400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00539__20210208.csv:2]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04089__20200210.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4089,CC_angular_resolution,1.076,
+4089,CC_depolarization_ratio,0.039,
+4089,CC_measurement_wavelength,700,
+4089,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04089__20200210.csv:5]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04101__20201013.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4101,CC_angular_resolution,1.076,
+4101,CC_depolarization_ratio,0.039,
+4101,CC_measurement_wavelength,700,
+4101,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04101__20201013.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01342__20201015.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1342,CC_angular_resolution,1.076,
+1342,CC_depolarization_ratio,0.039,
+1342,CC_measurement_wavelength,700,
+1342,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01342__20201015.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01347__20200306.csv [hunk 8]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1347,CC_angular_resolution,1.076,
+1347,CC_depolarization_ratio,0.039,
+1347,CC_measurement_wavelength,700,
+1347,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01347__20200306.csv:5]: looks good

File: calibration/PARADM/CGINS-PARADM-50202__20200225.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50202,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50202__20200225.csv:2]: looks good

File: calibration/PARADM/CGINS-PARADM-50217__20201103.csv [hunk 10]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50217,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50217__20201103.csv:2]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09295__20200217.csv (hunk 1 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09295__20200217.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09301__20201001.csv (hunk 2 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09301__20201001.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00432__20210208.csv (hunk 3 above)
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00432__20210208.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00539__20210208.csv (hunk 4 above)
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00539__20210208.csv:2]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-04089__20200210.csv (hunk 5 above)
Comment (snwhite) [reply calibration/FLORTM/CGINS-FLORTM-04089__20200210.csv:5]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-04101__20201013.csv (hunk 6 above)
Comment (snwhite) [reply calibration/FLORTM/CGINS-FLORTM-04101__20201013.csv:5]: Looks good.

File: calibration/FLORTO/CGINS-FLORTO-01342__20201015.csv (hunk 7 above)
Comment (snwhite) [reply calibration/FLORTO/CGINS-FLORTO-01342__20201015.csv:5]: These measure scattering at three wavelengths... not sure why we only have the 700 nm listed.

File: calibration/FLORTO/CGINS-FLORTO-01347__20200306.csv (hunk 8 above)
Comment (snwhite) [reply calibration/FLORTO/CGINS-FLORTO-01347__20200306.csv:5]: These measure scattering at three wavelengths... not sure why we only have the 700 nm listed.

File: calibration/PARADM/CGINS-PARADM-50202__20200225.csv (hunk 9 above)
Comment (snwhite) [reply calibration/PARADM/CGINS-PARADM-50202__20200225.csv:2]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50217__20201103.csv (hunk 10 above)
Comment (snwhite) [reply calibration/PARADM/CGINS-PARADM-50217__20201103.csv:2]: Looks good.

**Review by snwhite — APPROVED** [review]
All look good.

**Pull request merged**